okapi/services/fieldnotes/upload/docs.xml

```diff
@@ -0,0 +1,49 @@
+<xml>
+    <brief>Upload Fieldnotes</brief>
+    <issue-id>630</issue-id>
+    <desc>
+        <p>Upload a set of one or more fieldnotes records.</p>
```

Comment:
I think we can be more detailed here. Many developers (particularly the ones from countries running the OCPL branch) will not know what "fieldnotes" are. How about this:

This method allows you to upload a series of fieldnotes in the CSV format. Fieldnotes are draft versions of log entries. Once uploaded, users will be able to review, edit, and submit them via the Opencaching site.

Reply:
done